GliaWidgets/Sources/Theme/Theme.swift

```diff
@@ -91,8 +91,8 @@ public class Theme {
                     baseLight: config.globalColors?.baseLight.map { UIColor(hex: $0) },
                     baseDark: config.globalColors?.baseDark.map { UIColor(hex: $0) },
                     baseShade: config.globalColors?.baseShade.map { UIColor(hex: $0) },
-                    background: config.globalColors?.background.map { UIColor(hex: $0) },
```

Comment:
Why did you remove the background color?

Reply:
Thank you for the good comment. The background was removed, along with grey, because those colors do not belong to our palette (8 colors). It was added there by mistake, it seems like. Since baseLight and background are the same exact color, background was removed to align with our palette.